seatunnel-core/seatunnel-flink-starter/src/main/docker/Dockerfile

```diff
@@ -20,18 +20,29 @@ FROM $BASE_IMAGE
 
 ARG SCALA_VERSION
 ARG FLINK_VERSION
+ARG SEATUNNEL_VERSION
 
 RUN mkdir -p /flink ; cd /flink ; \
     tar_file=flink-${FLINK_VERSION}-bin-scala_${SCALA_VERSION}.tgz ; \
     curl -LsO https://archive.apache.org/dist/flink/flink-${FLINK_VERSION}/$tar_file ; \
     tar -zxf $tar_file --strip 1 -C . ; \
     rm $tar_file
-
+    
```

Comment:
revert？

Reply:
I don't get it. What should I revert exactly?